server/views/pages/prisonerList/macros/sortableTableHeader.njk

```diff
@@ -0,0 +1,44 @@
+{#  -------------------------------------------------------------
+    Sortable column header for use on Prisoner List page
+    `config` argument is an object with the following properties:
+      * searchTerm - the search term that the Prisoner List page is currently filtered by
+      * statusFilter - the value of the status filter dropdown that the Prisoner List page is currently filtered on
+      * sortBy - the fieldName that the Prisoner List is currently sorted by ( name | location | release-date | reception-date | status)
+      * sortOrder - the sort order that the Prisoner List is currently sorted on ( ascending | descending )
+      * fieldName - field name of this column ( name | location | release-date | reception-date | status)
+      * headerText - Display text for table header of this column
+    ------------------------------------------------------------- #}
+{% macro sortableTableHeader(config) %}
+
+  <th scope="col" class="govuk-table__header"
+      aria-sort="{{ config.sortOrder if config.sortBy === config.fieldName else 'none' }}" {# If this column header is the current sortBy field then use the current sort order value, else `none`#}
+      data-qa="{{ config.fieldName }}-column-header">
+
+    {# Set a variable defining the new sort order that will be used if this header is clicked.
+       If this column header is the current sortBy field then use the opposite of the current sort order.
+       If this column header is not the current sortBy field then use `ascending`
+    #}
+    {%- set newSortOrder -%}
+      {%- if config.sortBy !== config.fieldName -%}
+        ascending
+      {%- else -%}
+        {{- 'ascending' if config.sortOrder === 'descending' else 'descending' -}}
+      {%- endif -%}
+    {%- endset -%}
+
+    <button
```

Comment:
@chrimesdev , I need to check with your whether the aria label and non-visible text are correct .... easier to talk you through it to explain, so lets catch up after DSU

Reply:
Agreed that we will do this as part of https://dsdmoj.atlassian.net/browse/RR-465 

Left a comment on the JIRA ticket to this comment too